src/Controller/ExportController.php

```diff
@@ -95,9 +103,43 @@ public function export(Request $request, ?SalesChannelContext $context): Respons
             return $errorResponse;
         }
 
+        $excludeProductGroups = filter_var($request->get('excludeProductGroups'), FILTER_VALIDATE_BOOLEAN);
```

Comment:
You can also do `$request->query->getBoolean()`.

Reply:
I think these specific type methods are not available for older version.

Reply:
No this is related to the Symfony version, and the `getBoolean` method already exists since Symfony 2.6.
![symfony_vers](https://user-images.githubusercontent.com/20555112/134140972-eb89e826-e1eb-4aea-985e-9d5b5f19c1be.png)

* https://symfony.com/doc/2.8/components/http_foundation.html
* https://symfony.com/doc/current/components/http_foundation.html